lib/bolt/project_manager/config_migrator.rb

```diff
@@ -72,9 +72,10 @@ def migrate(config_file, project_file, inventory_file, backup_dir)
         data     = Bolt::Util.read_yaml_hash(project_file, 'config')
         modified = false
 
-        if data.key?('plugin_hooks') && !data.key?('plugin-hooks')
-          @outputter.print_action_step("Updating 'plugin_hooks' option to 'plugin-hooks'")
-          data['plugin-hooks'] = data.delete('plugin_hooks')
+        [%w[apply_settings apply-settings], %w[plugin_hooks plugin-hooks]].each do |old, new|
+          next unless data.key?(old) && !data.key?(new)
```

Comment:
If users have both should we just delete the old key? It will be ignored anyway.

Reply:
I thought about it with plugin-hooks and decided it was too destructive, but looking at it again I don't see a good reason *not* to do that.

Reply:
Updated to remove the deprecated option when both exist.